Tasks/AzureCLI/task.json

```diff
@@ -0,0 +1,104 @@
+{
+  "id": "46E4BE58-730B-4389-8A2F-EA10B3E5E815",
+  "name": "AzureCLI [Preview]",
+  "friendlyName": "Azure CLI preview",
+  "description": "Run a Shell script with Azure CLI commands against an azure subscription",
+  "author": "Microsoft Corporation",
+  "helpMarkDown": "Replace with markdown to show in help",
+  "category": "Utility",
+  "visibility":["Preview","Build","Release"],
+  "demands": [
+    "sh"
```

Comment:
This demand will not be met in Windows agents if this task has to be working for both Windows and Linux machines.